src/utils/fetch.spec.ts

```diff
@@ -360,6 +361,62 @@ describe('utils fetch', () => {
                 `);
     });
 
+    it('resolves if text type problem json', async () => {

```

Comment:
With `test.each` we could also make sure that other types are handled properly e.g.
it.each`
    contentType
    {"application/problem+json; charset=utf-8"}
    {"application/x-javascript"}
`('resolves if text type is $contentType', ({contentType}) => {......

Reply:
yeah I didn't think it necessary to test every case - just one where we are splitting it